### ISSUE TITLE
lr decay

### ISSUE THREAD

**Issue by WenmuZhou** [PaddlePaddle/PaddleOCR#58]
训练crnn，应该在哪里设置lr decay呢

**Comment by LDOUBLEV** [comment]
具体是在ppocr.optimizer.AdamDecay()函数中设置lr_decay。
不同的lr_decay方式可以参考[官网](https://paddlepaddle.org.cn/documentation/docs/zh/api_guides/low_level/layers/learning_rate_scheduler.html#api-guide-learning-rate-scheduler).

-----
PaddleOCR使用importlib 通过yml文件中的参数选择调用哪个模块，设置lr decay需要在优化器里，优化器的定义在program.py中的176行 
```
opt_params = config['Optimizer']
optimizer = create_module(opt_params['function'])(opt_params) 
```
opt_params 是读取到的yml文件中的参数，假设使用config/rec/rec_chinese_lite_train.yml配置文件，有关优化器的定义是：
```
Optimizer:
  function: ppocr.optimizer,AdamDecay
  base_lr: 0.0005
  beta1: 0.9
  beta2: 0.999
```
说明优化器的声明函数的是ppocr.optimizer.AdamDecay，所以可以在AdamDecay加上lr decay。
要修改其他超参数也是同理。

**Comment by WenmuZhou** [comment]
好的，谢谢
